frontend/uddhar-frontend/src/coordinator/components/TableWithPagination.jsx

```
@@ -167,22 +291,25 @@ const TableWithPagination = () => {
               disabled={currentPage === totalPages}
-              className="px-2 py-1 border rounded cursor-pointer"
+              className="px-2 py-1 border rounded cursor-pointer disabled:opacity-50"
             >
               &rsaquo;
             </button>
             <button
               onClick={() => handleClick(totalPages)}
               disabled={currentPage === totalPages}
-              className="px-2 py-1 border rounded cursor-pointer"
+              className="px-2 py-1 border rounded cursor-pointer disabled:opacity-50"
             >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)